bitnami/airflow/README.md

```diff
@@ -107,6 +107,9 @@ The following tables lists the configurable parameters of the Airflow chart and
 | `airflow.clonePluginsFromGit.branch`                                     | Branch from repository to checkout                                                                                                                                                                                             | `nil`                                                        |
 | `airflow.clonePluginsFromGit.path`                                       | Path to a folder in the repository containing the plugins.                                                                                                                                                                     | `nil`                                                        |
 | `airflow.baseUrl`                                                        | URL used to access to airflow web ui                                                                                                                                                                                           | `nil`                                                        |
+| `airflow.rbac.create`                                                    | Specifies whether RBAC rules should be created                                                                                                                                                                                 | `false`                                                      |
+| `airflow.serviceAccount`                                                 | Specifies whether a ServiceAccount should be created                                                                                                                                                                           | `false`                                                      |
```

Comment:
I would say this variable should be `airflow.serviceAccount.create`. Also, you forgot to add `airflow.serviceAccount.name`.

Reply:
Yes, my bad. Will fix, thanks.